docs/config.md

```diff
@@ -98,11 +98,20 @@ The services we currently support with links to their associated Lagoon docs is
 * [PostgreSQL](https://docs.lagoon.sh/lagoon/docker-images/postgres)
 * [Redis](https://docs.lagoon.sh/lagoon/docker-images/redis)
 * [Ruby](https://docs.lagoon.sh/docker-images/ruby/)
-* [Solr](https://docs.lagoon.sh/lagoon/docker-images/solr)
+* [Solr*](https://docs.lagoon.sh/lagoon/docker-images/solr)
 * [Varnish](https://docs.lagoon.sh/lagoon/docker-images/varnish)
 
 Note that we are testing against the "Drupal" variants of the above but it's _possible_ the base services work as well.
 
+*Please note: if using [uselagoon/solr-8-drupal](https://github.com/uselagoon/lagoon-images/blob/main/images/solr-drupal/8.Dockerfile) you will need to add the following command to your `Solr` service in your `.lando.yml` file.
```

Comment:
also mention that other solr images are supported here, and they may have different CMDs, this is just one example